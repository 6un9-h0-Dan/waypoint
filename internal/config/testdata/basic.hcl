project = "foo"

app "web" {
    build "pack" {
        registry "docker" {
            name = "gcr.io/mitchellh-test/myapp:latest"
        }
    }

    deploy "google-cloud-run" {}
}
