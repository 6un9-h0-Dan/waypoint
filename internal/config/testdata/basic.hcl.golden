(config.Config) {
 Apps: ([]*config.App) (len=1 cap=1) {
  (*config.App)({
   Name: (string) (len=3) "web",
   Path: (string) "",
   Build: (*config.Component)({
    Type: (string) (len=4) "pack",
    Body: (*hclsyntax.Body)({
     Attributes: (hclsyntax.Attributes) {
     },
     Blocks: (hclsyntax.Blocks) {
     },
     hiddenAttrs: (map[string]struct {}) {
     },
     hiddenBlocks: (map[string]struct {}) {
     },
     SrcRange: (hcl.Range) testdata/basic.hcl:2,18-20,
     EndRange: (hcl.Range) testdata/basic.hcl:2,20-20
    })
   }),
   Registry: (*config.Component)({
    Type: (string) (len=6) "docker",
    Body: (*hclsyntax.Body)({
     Attributes: (hclsyntax.Attributes) (len=1) {
      (string) (len=4) "name": (*hclsyntax.Attribute)({
       Name: (string) (len=4) "name",
       Expr: (*hclsyntax.TemplateExpr)({
        Parts: ([]hclsyntax.Expression) (len=1 cap=1) {
         (*hclsyntax.LiteralValueExpr)({
          Val: (cty.Value) {
           ty: (cty.Type) {
            typeImpl: (cty.primitiveType) {
             typeImplSigil: (cty.typeImplSigil) {
             },
             Kind: (cty.primitiveTypeKind) 83
            }
           },
           v: (string) (len=34) "gcr.io/mitchellh-test/myapp:latest"
          },
          SrcRange: (hcl.Range) testdata/basic.hcl:5,17-51
         })
        },
        SrcRange: (hcl.Range) testdata/basic.hcl:5,16-52
       }),
       SrcRange: (hcl.Range) testdata/basic.hcl:5,9-52,
       NameRange: (hcl.Range) testdata/basic.hcl:5,9-13,
       EqualsRange: (hcl.Range) testdata/basic.hcl:5,14-15
      })
     },
     Blocks: (hclsyntax.Blocks) {
     },
     hiddenAttrs: (map[string]struct {}) {
     },
     hiddenBlocks: (map[string]struct {}) {
     },
     SrcRange: (hcl.Range) testdata/basic.hcl:4,23-6,6,
     EndRange: (hcl.Range) testdata/basic.hcl:6,6-6
    })
   }),
   Platform: (*config.Component)({
    Type: (string) (len=16) "google-cloud-run",
    Body: (*hclsyntax.Body)({
     Attributes: (hclsyntax.Attributes) {
     },
     Blocks: (hclsyntax.Blocks) {
     },
     hiddenAttrs: (map[string]struct {}) {
     },
     hiddenBlocks: (map[string]struct {}) {
     },
     SrcRange: (hcl.Range) testdata/basic.hcl:8,31-33,
     EndRange: (hcl.Range) testdata/basic.hcl:8,33-33
    })
   })
  })
 }
}
